<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1299</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="grafikplot" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>481</width>
      <height>531</height>
     </rect>
    </property>
    <property name="toolTip">
     <string/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="11" column="0">
        <widget class="QPushButton" name="buttonclear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QPushButton" name="buttondelta">
         <property name="text">
          <string>Delta</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="Class2">
         <property name="text">
          <string>Class 2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="Class1">
         <property name="text">
          <string>Class 1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QCheckBox" name="checknorm">
         <property name="text">
          <string>Normalize</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>label2</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Perceptron</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>label1</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="labelcycle">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="8" column="0">
        <widget class="QPushButton" name="buttonDeltaMul">
         <property name="text">
          <string>Delta</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lableError">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="0">
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QPushButton" name="buttonCadd">
         <property name="text">
          <string>AddClass</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QPushButton" name="buttonPerceptronMul">
         <property name="text">
          <string>Perceptron</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QCheckBox" name="MultiNormalize">
         <property name="text">
          <string>Normalize</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelcycle_2">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>471</width>
        <height>501</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="buttondeltamultilayer">
         <property name="text">
          <string>Delta</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QCheckBox" name="checkBoxNormalizeMulti">
         <property name="text">
          <string>Normalize Data</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBoxMultiLayer"/>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="lineEditHiddenSize"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelHiddenSize">
         <property name="text">
          <string>Hidden Neuron Size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineEditClassSize"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelClasSize">
         <property name="text">
          <string>Class Size</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelCurrentSinif">
         <property name="text">
          <string>Sınıf</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QPushButton" name="pushButtonSet">
         <property name="text">
          <string>We Are Ready</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1299</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProcess">
    <property name="title">
     <string>Process</string>
    </property>
    <widget class="QMenu" name="menuinitial">
     <property name="title">
      <string>initial</string>
     </property>
     <addaction name="actionRandomly"/>
    </widget>
    <addaction name="menuinitial"/>
   </widget>
   <addaction name="menuProcess"/>
  </widget>
  <action name="actionRandomly">
   <property name="text">
    <string>Randomly</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
